### PR TITLE
feat(RHINENG-1658): Sort by group column

### PR DESCRIPTION
**Pull request by gkarat**
Implements https://issues.redhat.com/browse/RHINENG-1658, https://issues.redhat.com/browse/RHINENG-2694.

This enables sorting by the Group column at /inventory (the main systems view) and /groups/%id (the group details view, the Add systems modal).

## How to test

1. At /inventory and /groups/%id, verify that you are able to sort hosts by groups correctly in ascending and descending orders.

## Screenshots

image

image

### CODE REVIEW

**Review by bastilian — APPROVED** [review]
LGTM! Thank you @gkarat!

File: config/dev.webpack.config.js [hunk 1]
```diff
@@ -21,23 +21,6 @@ const { config: webpackConfig, plugins } = config({
           host: `http://localhost:${process.env.CONFIG_PORT}`,
         },
       }),
-      ...(process.env.LOCAL_API && {
```
Comment (vkrizan) [inline config/dev.webpack.config.js:24]: Why was this removed? Is it related to the feature?
